### PR TITLE
[OSDEV-536] Parse sector_product_type based on commas or vertical bars

### PR DESCRIPTION
**Pull request by mazursasha1990**
[OSDEV-536](https://opensupplyhub.atlassian.net/browse/OSDEV-536): Parse sector_product_type based on commas or vertical bars
In the `contricleaner` library, implement parsing of fields `sector_product_type`, `sector`, and `product_type` based on the `sector_split_pattern` passed to the `contricleaner` during calling. Currently, `sector_split_pattern` includes commas and vertical bars.

### CODE REVIEW

File: src/django/contricleaner/lib/serializers/row_serializers/row_sector_serializer.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def validate(self, row: dict, current: dict) -> dict:
             current["errors"].extend(sector_errors)
             return current
 
-        splitted_values = split_values(values, ', ')
+        splitted_values = split_values(values, r', |,|\|')
```
Comment (killev) [inline src/django/contricleaner/lib/serializers/row_serializers/row_sector_serializer.py:52]: Let's make possible to configure this values from Django i.e. pass them when we create Django parser...

Comment (mazursasha1990) [reply src/django/contricleaner/lib/serializers/row_serializers/row_sector_serializer.py]: Done

**Review by killev — APPROVED** [review]
LGTM, Approved

**Review by roman-stolar — APPROVED** [review]
LGTM

**Pull request merged**